Using increase_parallelism break the build

I wanted to try `opts.increase_parallelism(4)`, in doing so I got this error on a Mac platform:

```
 = note: Undefined symbols for architecture x86_64:
            "rocksdb_options_set_create_if_missing", referenced from:
                rocksdb::db_options::_$LT$impl$u20$rocksdb..Options$GT$::create_if_missing::h513249c3622a323c in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_options_increase_parallelism", referenced from:
                rocksdb::db_options::_$LT$impl$u20$rocksdb..Options$GT$::increase_parallelism::h2acca6293a0466de in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_close", referenced from:
                rocksdb::db::_$LT$impl$u20$core..ops..drop..Drop$u20$for$u20$rocksdb..DB$GT$::drop::h510736efbcc46707 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_readoptions_create", referenced from:
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::get::h6a4979bda91d799a in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_writebatch_create", referenced from:
                _$LT$rocksdb..db..WriteBatch$u20$as$u20$core..default..Default$GT$::default::h508a2a4c7c46b46a in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_writeoptions_create", referenced from:
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::write::hb9dfa1d84a27f2f5 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::put::h6ffaef0c5cae6783 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_writebatch_put", referenced from:
                rocksdb::db::WriteBatch::put::h7781b1f89e963f47 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_put", referenced from:
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::put::h6ffaef0c5cae6783 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_options_destroy", referenced from:
                rocksdb::db_options::_$LT$impl$u20$core..ops..drop..Drop$u20$for$u20$rocksdb..Options$GT$::drop::hbb4ddebe972fc22e in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_column_family_handle_destroy", referenced from:
                rocksdb::db::_$LT$impl$u20$core..ops..drop..Drop$u20$for$u20$rocksdb..DB$GT$::drop::h510736efbcc46707 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_get", referenced from:
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::get::h6a4979bda91d799a in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_write", referenced from:
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::write::hb9dfa1d84a27f2f5 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_writeoptions_destroy", referenced from:
                core::ptr::drop_in_place::h71c81480468bb47e in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::write::hb9dfa1d84a27f2f5 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::put::h6ffaef0c5cae6783 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_options_create", referenced from:
                rocksdb::db_options::_$LT$impl$u20$core..default..Default$u20$for$u20$rocksdb..Options$GT$::default::ha22de84a647edabd in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_writebatch_destroy", referenced from:
                core::ptr::drop_in_place::h0c27a3bfb535ae96 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::write::hb9dfa1d84a27f2f5 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
                _$LT$rocksdb..db..WriteBatch$u20$as$u20$core..ops..drop..Drop$GT$::drop::hd605aceb7fc5d220 in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_readoptions_destroy", referenced from:
                core::ptr::drop_in_place::h549810e89e82f67c in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::get::h6a4979bda91d799a in librocksdb-8d5ca9f0a97417b6.rlib(rocksdb-8d5ca9f0a97417b6.rocksdb0.rust-cgu.o)
            "rocksdb_open", referenced from:
                rocksdb::db::_$LT$impl$u20$rocksdb..DB$GT$::open::h07d8f9ec89aeda28 in txps-2be7e8a372e221f4.txps0.rust-cgu.o
                txps::main::h5a0ab24350e261c6 in txps-2be7e8a372e221f4.txps0.rust-cgu.o
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

```

--- a/tests/test_rocksdb_options.rs
+++ b/tests/test_rocksdb_options.rs
@@ -28,3 +28,14 @@ fn test_set_num_levels() {
         let _db = DB::open(&opts, &n).unwrap();
     }
 }
+
+#[test]
+fn test_increase_parallelism() {
+    let n = DBPath::new("_rust_rocksdb_test_increase_parallelism");
+    {
+        let mut opts = Options::default();
+        opts.create_if_missing(true);
+        opts.increase_parallelism(4);
+        let _db = DB::open(&opts, &n).unwrap();
+    }
+}

